util/mktar.sh

```diff
@@ -31,8 +31,7 @@ done
 if [ -z "$TARFILE" ]; then TARFILE="$NAME.tar"; fi
 
 # This counts on .gitattributes to specify what files should be ignored
-git archive --worktree-attributes --format=tar --prefix="$NAME/" -v HEAD \
-    | gzip -9 > "$TARFILE.gz"
+git archive -9 --prefix="$NAME/" -o $TARFILE.gz -v HEAD
```

Comment:
Hmmm, I disagree with the removal of `--worktree-attributes`, i.e. this tools should allow a developer to experiment with attributes without having to commit them.  The rest of the change is fine.

Reply:
> , i.e. this tools should allow a developer to experiment with attributes without having to commit them. 

True,  just please be noticed that  `--worktree-attributes` may also cause using the wrong `.gitattribute`  (someone update it without commit).